File: static/data/omphello.c
#include <stdio.h>
#include <omp.h>

int main() {

    int tid, nthreads;

#pragma omp parallel private(nthreads,tid)
    {
    nthreads=omp_get_num_threads();
    tid=omp_get_thread_num();
    printf("Hello from thread %d of %d\n",tid, nthreads);
    }
}

File: static/data/omphello.f90
program omp_hello
use omp_lib
implicit none

   integer :: i
   integer :: nthreads, tid

!$omp parallel private(nthreads,tid)
   nthreads=omp_get_num_threads()
   tid=omp_get_thread_num()
   print *, "Hello from thread ",tid,"of ",nthreads
!$omp end parallel

end program
